### PR TITLE
python3Packages.tinytag: init at 1.10.1

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/tinytag/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  preConfigure = ''
+    mkdir -p .github
+    touch .github/{README.md,LICENSE}
+  '';
+
+  pythonImportsCheck = [ "tinytag" ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = {
```
Comment (natsukium) [inline pkgs/development/python-modules/tinytag/default.nix:32]: Please add `meta.homepage`.

File: pkgs/development/python-modules/tinytag/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    rev = "refs/tags/${version}";
+    hash = "sha256-Kg67EwDIi/Io7KKnNiqPzQKginrfuE6FAeOCjFgEJkY=";
+  };
+
+  build-system = [ setuptools ];
+
+  preConfigure = ''
+    mkdir -p .github
+    touch .github/{README.md,LICENSE}
+  '';
```
Comment (natsukium) [inline pkgs/development/python-modules/tinytag/default.nix:26]: Why is this needed?

Comment (Sigmanificient) [reply pkgs/development/python-modules/tinytag/default.nix:26]: The build fails it thoses files aren't present

Comment (natsukium) [reply pkgs/development/python-modules/tinytag/default.nix:26]: I can't reproduce such an error.
Could you please check again?

Comment (Sigmanificient) [reply pkgs/development/python-modules/tinytag/default.nix]: Hmm that definitively not that case for `tinytag`, I think i got confused with `library`, which I was working on (tinytag is a dep of library)

**Comment by natsukium** [comment]
@ofborg build python312Packages.tinytag

I recommend that the commit message starts with `python312Packages` so that ofborg can automatically start the build.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

> Commit names of Python libraries should reflect that they are Python libraries, so write for example python311Packages.numpy: 1.11 -> 1.12. It is highly recommended to specify the current default version to enable automatic build by ofborg.

File: pkgs/development/python-modules/tinytag/default.nix (hunk 2 above)
Comment (natsukium) [reply pkgs/development/python-modules/tinytag/default.nix]: That package might have an unnecessary and strange build process.

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 331372` at a5541981 run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python311Packages.tinytag</li>
    <li>python312Packages.tinytag</li>
  </ul>
</details>

**Review by natsukium — APPROVED** [review]
Thank you!
